cli/cli.py

```diff
@@ -25,6 +35,44 @@ def list_resources(provider: str, resource_type: str):
         print_services(list_services(provider))
     elif resource_type == "fixers":
         print_fixers(list_fixers(provider))
+    elif resource_type == "categories":
+        print_categories(list_categories(bulk_load_checks_metadata(provider)))
+    elif resource_type == "compliance":
+        print_compliance_frameworks(bulk_load_compliance_frameworks(provider))
+    elif resource_type in ["checks", "checks-json"]:
+        bulk_checks_metadata = bulk_load_checks_metadata(provider)
+        checks_to_execute = load_checks_to_execute(
+            bulk_checks_metadata,
+            bulk_load_compliance_frameworks(provider),
+            None,
+            [],
+            [],
+            [],
+            [],
+            [],
+            provider,
+        )
+        if resource_type == "checks":
+            print_checks(provider, sorted(checks_to_execute), bulk_checks_metadata)
+        elif resource_type == "checks-json":
+            print(list_checks_json(provider, sorted(checks_to_execute)))
+
+
+def list_compliance_requirements(
+    provider: str, compliance_frameworks: list[str] = None
+):
+    print_compliance_requirements(
+        bulk_load_compliance_frameworks(provider), compliance_frameworks
+    )
+
+
+def validate_frameworks(compliance_frameworks: list[str] = None):
+    if not compliance_frameworks:
+        raise typer.BadParameter("Expected at least one argument")
+    for framework in compliance_frameworks:
+        if framework not in available_compliance_frameworks:
+            print(f"{framework} is not a valid Compliance Framework\n")
+    return compliance_frameworks
 
 
 def create_list_commands(provider_typer: typer.Typer):
```

Comment:
The purpose of this function is to define another set of functions for Typer to get these commands, am I right? Is this a recommended pattern?

Reply:
It would be used this way, currently it's just to check that the compliance specified are correct